<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en"><head>

<meta charset="utf-8">
<meta name="generator" content="quarto-1.1.189">

<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">


<title>elliptical.bifaces - GM: Raw Material</title>
<style>
code{white-space: pre-wrap;}
span.smallcaps{font-variant: small-caps;}
div.columns{display: flex; gap: min(4vw, 1.5em);}
div.column{flex: auto; overflow-x: auto;}
div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
ul.task-list{list-style: none;}
ul.task-list li input[type="checkbox"] {
  width: 0.8em;
  margin: 0 0.8em 0.2em -1.6em;
  vertical-align: middle;
}
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
div.sourceCode { margin: 1em 0; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>


<script src="site_libs/quarto-nav/quarto-nav.js"></script>
<script src="site_libs/quarto-nav/headroom.min.js"></script>
<script src="site_libs/clipboard/clipboard.min.js"></script>
<script src="site_libs/quarto-search/autocomplete.umd.js"></script>
<script src="site_libs/quarto-search/fuse.min.js"></script>
<script src="site_libs/quarto-search/quarto-search.js"></script>
<meta name="quarto:offset" content="./">
<script src="site_libs/quarto-html/quarto.js"></script>
<script src="site_libs/quarto-html/popper.min.js"></script>
<script src="site_libs/quarto-html/tippy.umd.min.js"></script>
<script src="site_libs/quarto-html/anchor.min.js"></script>
<link href="site_libs/quarto-html/tippy.css" rel="stylesheet">
<link href="site_libs/quarto-html/quarto-syntax-highlighting.css" rel="stylesheet" id="quarto-text-highlighting-styles">
<script src="site_libs/bootstrap/bootstrap.min.js"></script>
<link href="site_libs/bootstrap/bootstrap-icons.css" rel="stylesheet">
<link href="site_libs/bootstrap/bootstrap.min.css" rel="stylesheet" id="quarto-bootstrap" data-mode="light">
<script id="quarto-search-options" type="application/json">{
  "location": "navbar",
  "copy-button": false,
  "collapse-after": 3,
  "panel-placement": "end",
  "type": "overlay",
  "limit": 20,
  "language": {
    "search-no-results-text": "No results",
    "search-matching-documents-text": "matching documents",
    "search-copy-link-title": "Copy link to search",
    "search-hide-matches-text": "Hide additional matches",
    "search-more-match-text": "more match in this document",
    "search-more-matches-text": "more matches in this document",
    "search-clear-button-title": "Clear",
    "search-detached-cancel-button-title": "Cancel",
    "search-submit-button-title": "Submit"
  }
}</script>


<link rel="stylesheet" href="styles.css">
</head>

<body class="nav-fixed">

<div id="quarto-search-results"></div>
  <header id="quarto-header" class="headroom fixed-top">
    <nav class="navbar navbar-expand-lg navbar-dark ">
      <div class="navbar-container container-fluid">
      <div class="navbar-brand-container">
    <a class="navbar-brand" href="./index.html">
    <span class="navbar-title">elliptical.bifaces</span>
    </a>
  </div>
          <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarCollapse" aria-controls="navbarCollapse" aria-expanded="false" aria-label="Toggle navigation" onclick="if (window.quartoToggleHeadroom) { window.quartoToggleHeadroom(); }">
  <span class="navbar-toggler-icon"></span>
</button>
          <div class="collapse navbar-collapse" id="navbarCollapse">
            <ul class="navbar-nav navbar-nav-scroll me-auto">
  <li class="nav-item">
    <a class="nav-link" href="./index.html">Home</a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./sizeclass.html">Define Size classes</a>
  </li>  
  <li class="nav-item">
    <a class="nav-link active" href="./gmraw.html" aria-current="page">GM: Raw Material</a>
  </li>  
  <li class="nav-item">
    <a class="nav-link" href="./gmszclass.html">GM: Size Class</a>
  </li>  
</ul>
            <ul class="navbar-nav navbar-nav-scroll ms-auto">
  <li class="nav-item compact">
    <a class="nav-link" href="https://github.com/seldenlab/elliptical.bifaces"><i class="bi bi-github" role="img" aria-label="GitHub">
</i> 
 </a>
  </li>  
</ul>
              <div id="quarto-search" class="" title="Search"></div>
          </div> <!-- /navcollapse -->
      </div> <!-- /container-fluid -->
    </nav>
</header>
<!-- content -->
<div id="quarto-content" class="quarto-container page-columns page-rows-contents page-layout-article page-navbar">
<!-- sidebar -->
<!-- margin-sidebar -->
    <div id="quarto-margin-sidebar" class="sidebar margin-sidebar">
        <nav id="TOC" role="doc-toc" class="toc-active">
    <h2 id="toc-title">On this page</h2>
   
  <ul>
  <li><a href="#load-packages-data" id="toc-load-packages-data" class="nav-link active" data-scroll-target="#load-packages-data">Load packages + data</a></li>
  <li><a href="#generalised-procrustes-analysis" id="toc-generalised-procrustes-analysis" class="nav-link" data-scroll-target="#generalised-procrustes-analysis">Generalised Procrustes Analysis</a></li>
  <li><a href="#principal-components-analysis" id="toc-principal-components-analysis" class="nav-link" data-scroll-target="#principal-components-analysis">Principal Components Analysis</a>
  <ul class="collapse">
  <li><a href="#minimamaxima-of-pc12-with-warp-grids" id="toc-minimamaxima-of-pc12-with-warp-grids" class="nav-link" data-scroll-target="#minimamaxima-of-pc12-with-warp-grids">Minima/maxima of PC1/2 with warp grids</a></li>
  </ul></li>
  <li><a href="#procrustes-anova-allometry" id="toc-procrustes-anova-allometry" class="nav-link" data-scroll-target="#procrustes-anova-allometry">Procrustes ANOVA: Allometry</a></li>
  <li><a href="#procrustes-anova" id="toc-procrustes-anova" class="nav-link" data-scroll-target="#procrustes-anova">Procrustes ANOVA</a></li>
  <li><a href="#morphological-disparity" id="toc-morphological-disparity" class="nav-link" data-scroll-target="#morphological-disparity">Morphological Disparity</a></li>
  </ul>
</nav>
    </div>
<!-- main -->
<main class="content" id="quarto-document-content">

<header id="title-block-header" class="quarto-title-block default">
<div class="quarto-title">
<h1 class="title">GM: Raw Material</h1>
</div>



<div class="quarto-title-meta">

    
    
  </div>
  

</header>

<p>Bifaces from Millsap Cache and Jowell Farm were used to assess morphological stability/variability in elliptical bifaces by site/raw material. Variability in elliptical biface morphology would provide evidence of local differences in morphology as a function of geography/raw material.</p>
<p>This analysis follows recent formal analyses of Perdiz arrow points <span class="citation" data-cites="RN9551 RN11264">(<a href="#ref-RN9551" role="doc-biblioref">Selden Jr et al. 2021</a>; <a href="#ref-RN11264" role="doc-biblioref">Selden Jr. 2022</a>)</span> as well as another that is in press (<a href="https://github.com/seldenlab/perdiz3/blob/main/ms/perdiz3.pdf">Selden and Dockall, in press</a>), and Gahagan biface morphology across the southern Caddo area and central Texas <span class="citation" data-cites="RN11783 RN21001 RN11264">(<a href="#ref-RN11783" role="doc-biblioref">Selden Jr., Dockall, and Shafer 2018</a>; <a href="#ref-RN21001" role="doc-biblioref">Selden Jr., Dockall, and Dubied 2020</a>; <a href="#ref-RN11264" role="doc-biblioref">Selden Jr. 2022</a>)</span>, where Perdiz arrow points and Gahagan bifaces have been found to differ in shape across the same geography as Caddo bottles <span class="citation" data-cites="RN11801 RN11782 RN11716 RN8312">(<a href="#ref-RN11801" role="doc-biblioref">Selden Jr. 2018a</a>, <a href="#ref-RN11782" role="doc-biblioref">2018b</a>, <a href="#ref-RN11716" role="doc-biblioref">2019</a>, <a href="#ref-RN8312" role="doc-biblioref">2021</a>)</span>. This analysis also follows another geometric morphometric analysis in which Perdiz arrow points from the ancestral Caddo area were found to differ as a function of <em>time</em>, <em>raw material</em>, and <em>burial context</em> <span class="citation" data-cites="RN9551">(<a href="#ref-RN9551" role="doc-biblioref">Selden Jr et al. 2021</a>)</span>.</p>
<section id="load-packages-data" class="level2">
<h2 class="anchored" data-anchor-id="load-packages-data">Load packages + data</h2>
<div class="cell">
<div class="sourceCode cell-code" id="cb1"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb1-1"><a href="#cb1-1" aria-hidden="true" tabindex="-1"></a><span class="co"># load required analysis packages</span></span>
<span id="cb1-2"><a href="#cb1-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(here)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>here() starts at E:/github/elliptical.bifaces/supp</code></pre>
</div>
<div class="sourceCode cell-code" id="cb3"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(StereoMorph)</span>
<span id="cb3-2"><a href="#cb3-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(geomorph)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading required package: RRPP</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading required package: rgl</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>Loading required package: Matrix</code></pre>
</div>
<div class="sourceCode cell-code" id="cb7"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb7-1"><a href="#cb7-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(tidyverse)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stderr">
<pre><code>── Attaching packages
───────────────────────────────────────
tidyverse 1.3.2 ──</code></pre>
</div>
<div class="cell-output cell-output-stderr">
<pre><code>✔ ggplot2 3.3.6      ✔ purrr   0.3.5 
✔ tibble  3.1.8      ✔ dplyr   1.0.10
✔ tidyr   1.2.1      ✔ stringr 1.4.1 
✔ readr   2.1.3      ✔ forcats 0.5.2 
── Conflicts ────────────────────────────────────────── tidyverse_conflicts() ──
✖ tidyr::expand() masks Matrix::expand()
✖ dplyr::filter() masks stats::filter()
✖ dplyr::lag()    masks stats::lag()
✖ tidyr::pack()   masks Matrix::pack()
✖ tidyr::unpack() masks Matrix::unpack()</code></pre>
</div>
<div class="sourceCode cell-code" id="cb10"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb10-1"><a href="#cb10-1" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(wesanderson)</span>
<span id="cb10-2"><a href="#cb10-2" aria-hidden="true" tabindex="-1"></a><span class="fu">library</span>(ggplot2)</span>
<span id="cb10-3"><a href="#cb10-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-4"><a href="#cb10-4" aria-hidden="true" tabindex="-1"></a><span class="co"># read data and define number of sLMs ----</span></span>
<span id="cb10-5"><a href="#cb10-5" aria-hidden="true" tabindex="-1"></a>shapes <span class="ot">&lt;-</span> <span class="fu">readShapes</span>(<span class="st">"shapes"</span>)</span>
<span id="cb10-6"><a href="#cb10-6" aria-hidden="true" tabindex="-1"></a>shapesGM <span class="ot">&lt;-</span> <span class="fu">readland.shapes</span>(shapes, <span class="at">nCurvePts =</span> <span class="fu">c</span>(<span class="dv">15</span>, <span class="dv">20</span>, <span class="dv">35</span>))</span>
<span id="cb10-7"><a href="#cb10-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb10-8"><a href="#cb10-8" aria-hidden="true" tabindex="-1"></a><span class="co"># read qualitative data</span></span>
<span id="cb10-9"><a href="#cb10-9" aria-hidden="true" tabindex="-1"></a>qdata <span class="ot">&lt;-</span> <span class="fu">read.csv</span>(<span class="st">"qdata.merged.csv"</span>, </span>
<span id="cb10-10"><a href="#cb10-10" aria-hidden="true" tabindex="-1"></a>                  <span class="at">header =</span> <span class="cn">TRUE</span>, </span>
<span id="cb10-11"><a href="#cb10-11" aria-hidden="true" tabindex="-1"></a>                  <span class="at">row.names =</span> <span class="dv">1</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="generalised-procrustes-analysis" class="level2">
<h2 class="anchored" data-anchor-id="generalised-procrustes-analysis">Generalised Procrustes Analysis</h2>
<p>Landmark data were aligned to a global coordinate system <span class="citation" data-cites="RN8477 RN7502 RN11622 RN11623 RN11563">(<a href="#ref-RN8477" role="doc-biblioref">Bookstein et al. 1999</a>; <a href="#ref-RN7502" role="doc-biblioref">Gunz, Mitteroecker, and Bookstein 2005</a>; <a href="#ref-RN11622" role="doc-biblioref">Kendall 1981</a>, <a href="#ref-RN11623" role="doc-biblioref">1984</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>, achieved through generalised Procrustes superimposition <span class="citation" data-cites="RN11138 RN478 RN1646">(<a href="#ref-RN11138" role="doc-biblioref">Bookstein 1986</a>; <a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>; <a href="#ref-RN1646" role="doc-biblioref">Rohlf 1999</a>)</span> in R using the <code>geomorph</code> and <code>RRPP</code> packages <span class="citation" data-cites="RN1655 RN11775 RN11530 RN1774 RN9565">(<a href="#ref-RN1655" role="doc-biblioref">Adams and Collyer 2015</a>; <a href="#ref-RN11775" role="doc-biblioref">Collyer and Adams 2018</a>; <a href="#ref-RN11530" role="doc-biblioref">Adams et al. 2017</a>; <a href="#ref-RN1774" role="doc-biblioref">Adams and Otarola-Castillo 2013</a>; <a href="#ref-RN9565" role="doc-biblioref">Baken et al. 2021</a>)</span>. Procrustes superimposition translates and rotates the coordinate data to allow for comparisons among objects, while also scaling each biface using unit-centroid size—the square root of the sum of squared distances from each landmark to the specimen’s centroid <span class="citation" data-cites="RN11139 RN11140 RN11564 RN478">(<a href="#ref-RN11139" role="doc-biblioref">Chapman 1990</a>; <a href="#ref-RN11140" role="doc-biblioref">Dryden and Mardia 1998</a>; <a href="#ref-RN11564" role="doc-biblioref">Gower 1975</a>; <a href="#ref-RN478" role="doc-biblioref">Rohlf and Slice 1990</a>)</span>. The <code>geomorph</code> package uses a partial Procrustes superimposition that projects the aligned specimens into tangent space subsequent to alignment in preparation for the use of multivariate methods that assume linear space <span class="citation" data-cites="RN11141 RN11142 RN1646 RN11563">(<a href="#ref-RN11141" role="doc-biblioref">Dryden and Mardia 1993</a>; <a href="#ref-RN11142" role="doc-biblioref">Kent and Mardia 2001</a>; <a href="#ref-RN1646" role="doc-biblioref">Rohlf 1999</a>; <a href="#ref-RN11563" role="doc-biblioref">Slice 2001</a>)</span>.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb11"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb11-1"><a href="#cb11-1" aria-hidden="true" tabindex="-1"></a><span class="co"># gpa ----</span></span>
<span id="cb11-2"><a href="#cb11-2" aria-hidden="true" tabindex="-1"></a>Y.gpa <span class="ot">&lt;-</span> <span class="fu">gpagen</span>(shapesGM, <span class="at">print.progress =</span> <span class="cn">FALSE</span>)</span>
<span id="cb11-3"><a href="#cb11-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-4"><a href="#cb11-4" aria-hidden="true" tabindex="-1"></a><span class="co"># geomorph data frame ----</span></span>
<span id="cb11-5"><a href="#cb11-5" aria-hidden="true" tabindex="-1"></a>gdf <span class="ot">&lt;-</span> <span class="fu">geomorph.data.frame</span>(<span class="at">shape =</span> Y.gpa<span class="sc">$</span>coords, </span>
<span id="cb11-6"><a href="#cb11-6" aria-hidden="true" tabindex="-1"></a>                           <span class="at">size =</span> Y.gpa<span class="sc">$</span>Csize,</span>
<span id="cb11-7"><a href="#cb11-7" aria-hidden="true" tabindex="-1"></a>                           <span class="at">site =</span> qdata<span class="sc">$</span>site)</span>
<span id="cb11-8"><a href="#cb11-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb11-9"><a href="#cb11-9" aria-hidden="true" tabindex="-1"></a><span class="co"># add centroid size to qdata</span></span>
<span id="cb11-10"><a href="#cb11-10" aria-hidden="true" tabindex="-1"></a>qdata<span class="sc">$</span>csz <span class="ot">&lt;-</span> Y.gpa<span class="sc">$</span>Csize</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
</div>
</section>
<section id="principal-components-analysis" class="level2">
<h2 class="anchored" data-anchor-id="principal-components-analysis">Principal Components Analysis</h2>
<p>Principal components analysis <span class="citation" data-cites="RN1746">(<a href="#ref-RN1746" role="doc-biblioref">Jolliffe 2002</a>)</span> was used to visualise shape variation among the elliptical bifaces, and the scatterplot represents the dispersion of shapes in tangent space <span class="citation" data-cites="RN8633 RN5616 RN11143 RN7550">(<a href="#ref-RN8633" role="doc-biblioref">Adams, Rohlf, and Slice 2013</a>; <a href="#ref-RN5616" role="doc-biblioref">Bookstein 1991</a>; <a href="#ref-RN11143" role="doc-biblioref">O’Higgins and Jones 1998</a>; <a href="#ref-RN7550" role="doc-biblioref">Rohlf and Marcus 1993</a>)</span>. Shape ranges described by each principal axis are commonly visualized using thin-plate spline warping of a reference image <span class="citation" data-cites="RN1731 RN479">(<a href="#ref-RN1731" role="doc-biblioref">Klingenberg 2013</a>; <a href="#ref-RN479" role="doc-biblioref">Sherratt et al. 2014</a>)</span>.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb12"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb12-1"><a href="#cb12-1" aria-hidden="true" tabindex="-1"></a><span class="co"># principal components analysis ----</span></span>
<span id="cb12-2"><a href="#cb12-2" aria-hidden="true" tabindex="-1"></a>pca<span class="ot">&lt;-</span><span class="fu">gm.prcomp</span>(Y.gpa<span class="sc">$</span>coords)</span>
<span id="cb12-3"><a href="#cb12-3" aria-hidden="true" tabindex="-1"></a><span class="fu">summary</span>(pca)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Ordination type: Principal Component Analysis 
Centering by OLS mean
Orthogonal projection of OLS residuals
Number of observations: 30 
Number of vectors 29 

Importance of Components:
                             Comp1        Comp2        Comp3       Comp4
Eigenvalues            0.007937426 0.0004428972 0.0003168109 0.000068305
Proportion of Variance 0.897778529 0.0500947764 0.0358335384 0.007725774
Cumulative Proportion  0.897778529 0.9478733056 0.9837068440 0.991432619
                              Comp5        Comp6        Comp7        Comp8
Eigenvalues            2.609697e-05 1.412387e-05 1.070065e-05 5.974135e-06
Proportion of Variance 2.951751e-03 1.597508e-03 1.210319e-03 6.757166e-04
Cumulative Proportion  9.943844e-01 9.959819e-01 9.971922e-01 9.978679e-01
                              Comp9       Comp10       Comp11       Comp12
Eigenvalues            5.511185e-06 3.520416e-06 2.843079e-06 1.882903e-06
Proportion of Variance 6.233537e-04 3.981838e-04 3.215721e-04 2.129696e-04
Cumulative Proportion  9.984913e-01 9.988895e-01 9.992110e-01 9.994240e-01
                             Comp13       Comp14       Comp15       Comp16
Eigenvalues            1.338169e-06 1.008447e-06 8.719528e-07 5.221444e-07
Proportion of Variance 1.513563e-04 1.140624e-04 9.862399e-05 5.905820e-05
Cumulative Proportion  9.995753e-01 9.996894e-01 9.997880e-01 9.998471e-01
                             Comp17       Comp18       Comp19       Comp20
Eigenvalues            3.798391e-07 3.058797e-07 1.881087e-07 1.054837e-07
Proportion of Variance 4.296247e-05 3.459714e-05 2.127642e-05 1.193095e-05
Cumulative Proportion  9.998901e-01 9.999247e-01 9.999459e-01 9.999579e-01
                             Comp21       Comp22       Comp23       Comp24
Eigenvalues            1.000897e-07 8.436762e-08 5.669035e-08 4.791035e-08
Proportion of Variance 1.132084e-05 9.542570e-06 6.412077e-06 5.418997e-06
Cumulative Proportion  9.999692e-01 9.999787e-01 9.999851e-01 9.999906e-01
                             Comp25       Comp26       Comp27       Comp28
Eigenvalues            2.911181e-08 2.405157e-08 1.247071e-08 1.033102e-08
Proportion of Variance 3.292750e-06 2.720401e-06 1.410525e-06 1.168511e-06
Cumulative Proportion  9.999938e-01 9.999966e-01 9.999980e-01 9.999991e-01
                             Comp29
Eigenvalues            7.543288e-09
Proportion of Variance 8.531988e-07
Cumulative Proportion  1.000000e+00</code></pre>
</div>
<div class="sourceCode cell-code" id="cb14"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb14-1"><a href="#cb14-1" aria-hidden="true" tabindex="-1"></a><span class="co"># set plot parameters</span></span>
<span id="cb14-2"><a href="#cb14-2" aria-hidden="true" tabindex="-1"></a>pch.gps <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="dv">1</span><span class="sc">:</span><span class="dv">2</span>)[<span class="fu">as.factor</span>(qdata<span class="sc">$</span>site)]</span>
<span id="cb14-3"><a href="#cb14-3" aria-hidden="true" tabindex="-1"></a>col.gps <span class="ot">&lt;-</span> <span class="fu">wes_palette</span>(<span class="st">"Moonrise2"</span>)[<span class="fu">as.factor</span>(qdata<span class="sc">$</span>site)]</span>
<span id="cb14-4"><a href="#cb14-4" aria-hidden="true" tabindex="-1"></a>col.hull <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="st">"#C27D38"</span>, <span class="st">"#798E87"</span>)</span>
<span id="cb14-5"><a href="#cb14-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb14-6"><a href="#cb14-6" aria-hidden="true" tabindex="-1"></a><span class="co"># plot pca by merged</span></span>
<span id="cb14-7"><a href="#cb14-7" aria-hidden="true" tabindex="-1"></a>pc.plot <span class="ot">&lt;-</span> <span class="fu">plot</span>(pca, </span>
<span id="cb14-8"><a href="#cb14-8" aria-hidden="true" tabindex="-1"></a>                <span class="at">asp =</span> <span class="dv">1</span>,</span>
<span id="cb14-9"><a href="#cb14-9" aria-hidden="true" tabindex="-1"></a>                <span class="at">pch =</span> pch.gps,</span>
<span id="cb14-10"><a href="#cb14-10" aria-hidden="true" tabindex="-1"></a>                <span class="at">col =</span> col.gps)</span>
<span id="cb14-11"><a href="#cb14-11" aria-hidden="true" tabindex="-1"></a><span class="fu">shapeHulls</span>(pc.plot, </span>
<span id="cb14-12"><a href="#cb14-12" aria-hidden="true" tabindex="-1"></a>           <span class="at">groups =</span> qdata<span class="sc">$</span>site,</span>
<span id="cb14-13"><a href="#cb14-13" aria-hidden="true" tabindex="-1"></a>           <span class="at">group.cols =</span> col.hull)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="gmraw_files/figure-html/pca.plotsiteraw-1.png" class="img-fluid" style="width:100.0%"></p>
</div>
</div>
<section id="minimamaxima-of-pc12-with-warp-grids" class="level3">
<h3 class="anchored" data-anchor-id="minimamaxima-of-pc12-with-warp-grids">Minima/maxima of PC1/2 with warp grids</h3>
<div class="cell">
<div class="sourceCode cell-code" id="cb15"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb15-1"><a href="#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="co"># plot x/y maxima/minima</span></span>
<span id="cb15-2"><a href="#cb15-2" aria-hidden="true" tabindex="-1"></a><span class="do">## x - minima</span></span>
<span id="cb15-3"><a href="#cb15-3" aria-hidden="true" tabindex="-1"></a>mean.shape <span class="ot">&lt;-</span> <span class="fu">mshape</span>(Y.gpa<span class="sc">$</span>coords)</span>
<span id="cb15-4"><a href="#cb15-4" aria-hidden="true" tabindex="-1"></a><span class="fu">plotRefToTarget</span>(pca<span class="sc">$</span>shapes<span class="sc">$</span>shapes.comp1<span class="sc">$</span>min, </span>
<span id="cb15-5"><a href="#cb15-5" aria-hidden="true" tabindex="-1"></a>                mean.shape)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="gmraw_files/figure-html/minmaxraw-1.png" class="img-fluid" style="width:100.0%"></p>
</div>
<div class="sourceCode cell-code" id="cb16"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb16-1"><a href="#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="do">## x - maxima</span></span>
<span id="cb16-2"><a href="#cb16-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plotRefToTarget</span>(pca<span class="sc">$</span>shapes<span class="sc">$</span>shapes.comp1<span class="sc">$</span>max, </span>
<span id="cb16-3"><a href="#cb16-3" aria-hidden="true" tabindex="-1"></a>                mean.shape)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="gmraw_files/figure-html/minmaxraw-2.png" class="img-fluid" style="width:100.0%"></p>
</div>
<div class="sourceCode cell-code" id="cb17"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb17-1"><a href="#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="do">## y - minima</span></span>
<span id="cb17-2"><a href="#cb17-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plotRefToTarget</span>(pca<span class="sc">$</span>shapes<span class="sc">$</span>shapes.comp2<span class="sc">$</span>min, </span>
<span id="cb17-3"><a href="#cb17-3" aria-hidden="true" tabindex="-1"></a>                mean.shape)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="gmraw_files/figure-html/minmaxraw-3.png" class="img-fluid" style="width:100.0%"></p>
</div>
<div class="sourceCode cell-code" id="cb18"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb18-1"><a href="#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="do">## y - maxima</span></span>
<span id="cb18-2"><a href="#cb18-2" aria-hidden="true" tabindex="-1"></a><span class="fu">plotRefToTarget</span>(pca<span class="sc">$</span>shapes<span class="sc">$</span>shapes.comp2<span class="sc">$</span>max, </span>
<span id="cb18-3"><a href="#cb18-3" aria-hidden="true" tabindex="-1"></a>                mean.shape)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output-display">
<p><img src="gmraw_files/figure-html/minmaxraw-4.png" class="img-fluid" style="width:100.0%"></p>
</div>
</div>
</section>
</section>
<section id="procrustes-anova-allometry" class="level2">
<h2 class="anchored" data-anchor-id="procrustes-anova-allometry">Procrustes ANOVA: Allometry</h2>
<p>To assess whether shape changes with size (allometry), Procrustes ANOVAs <span class="citation" data-cites="RN1749">(<a href="#ref-RN1749" role="doc-biblioref">Goodall 1991</a>)</span> were run that enlist effect-sizes (zscores) computed as standard deviates of the generated sampling distributions <span class="citation" data-cites="RN1756">(<a href="#ref-RN1756" role="doc-biblioref">Collyer, Sekora, and Adams 2015</a>)</span>. A residual randomization permutation procedure (RRPP; n = 10,000 permutations) was used for all Procrustes ANOVAs <span class="citation" data-cites="RN1655 RN11775">(<a href="#ref-RN1655" role="doc-biblioref">Adams and Collyer 2015</a>; <a href="#ref-RN11775" role="doc-biblioref">Collyer and Adams 2018</a>)</span>, which has higher statistical power and a greater ability to identify patterns in the data should they be present <span class="citation" data-cites="RN1719">(<a href="#ref-RN1719" role="doc-biblioref">Anderson and Ter Braak 2003</a>)</span>.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb19"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb19-1"><a href="#cb19-1" aria-hidden="true" tabindex="-1"></a><span class="co"># allometry</span></span>
<span id="cb19-2"><a href="#cb19-2" aria-hidden="true" tabindex="-1"></a>fit.size <span class="ot">&lt;-</span> <span class="fu">procD.lm</span>(shape <span class="sc">~</span> size, </span>
<span id="cb19-3"><a href="#cb19-3" aria-hidden="true" tabindex="-1"></a>                     <span class="at">data =</span> gdf, </span>
<span id="cb19-4"><a href="#cb19-4" aria-hidden="true" tabindex="-1"></a>                     </span>
<span id="cb19-5"><a href="#cb19-5" aria-hidden="true" tabindex="-1"></a>                     <span class="at">print.progress =</span> <span class="cn">FALSE</span>, </span>
<span id="cb19-6"><a href="#cb19-6" aria-hidden="true" tabindex="-1"></a>                     <span class="at">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb19-7"><a href="#cb19-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb19-8"><a href="#cb19-8" aria-hidden="true" tabindex="-1"></a><span class="co"># shape change with size?</span></span>
<span id="cb19-9"><a href="#cb19-9" aria-hidden="true" tabindex="-1"></a><span class="fu">anova</span>(fit.size)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Analysis of Variance, using Residual Randomization
Permutation procedure: Randomization of null model residuals 
Number of permutations: 10000 
Estimation method: Ordinary Least Squares 
Sums of Squares and Cross-products: Type I 
Effect sizes (Z) based on F distributions

          Df       SS        MS     Rsq      F        Z Pr(&gt;F)
size       1 0.003113 0.0031127 0.01214 0.3441 -0.36128 0.6234
Residuals 28 0.253282 0.0090458 0.98786                       
Total     29 0.256394                                         

Call: procD.lm(f1 = shape ~ size, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
</div>
</div>
</section>
<section id="procrustes-anova" class="level2">
<h2 class="anchored" data-anchor-id="procrustes-anova">Procrustes ANOVA</h2>
<p>To assess whether shape and size differ by <code>size class</code>, Procrustes ANOVAs <span class="citation" data-cites="RN1749">(<a href="#ref-RN1749" role="doc-biblioref">Goodall 1991</a>)</span> were run that enlist effect-sizes (zscores) computed as standard deviates of the generated sampling distributions <span class="citation" data-cites="RN1756">(<a href="#ref-RN1756" role="doc-biblioref">Collyer, Sekora, and Adams 2015</a>)</span>. A residual randomization permutation procedure (RRPP; n = 10,000 permutations) was used for all Procrustes ANOVAs <span class="citation" data-cites="RN1655 RN11775">(<a href="#ref-RN1655" role="doc-biblioref">Adams and Collyer 2015</a>; <a href="#ref-RN11775" role="doc-biblioref">Collyer and Adams 2018</a>)</span>, which has higher statistical power and a greater ability to identify patterns in the data should they be present <span class="citation" data-cites="RN1719">(<a href="#ref-RN1719" role="doc-biblioref">Anderson and Ter Braak 2003</a>)</span>.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb21"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb21-1"><a href="#cb21-1" aria-hidden="true" tabindex="-1"></a><span class="co"># ANOVA ----</span></span>
<span id="cb21-2"><a href="#cb21-2" aria-hidden="true" tabindex="-1"></a><span class="co"># shape</span></span>
<span id="cb21-3"><a href="#cb21-3" aria-hidden="true" tabindex="-1"></a>fit.sh <span class="ot">&lt;-</span> <span class="fu">procD.lm</span>(shape <span class="sc">~</span> site,</span>
<span id="cb21-4"><a href="#cb21-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">data =</span> gdf,</span>
<span id="cb21-5"><a href="#cb21-5" aria-hidden="true" tabindex="-1"></a>                   <span class="at">print.progress =</span> <span class="cn">FALSE</span>,</span>
<span id="cb21-6"><a href="#cb21-6" aria-hidden="true" tabindex="-1"></a>                   <span class="at">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb21-7"><a href="#cb21-7" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb21-8"><a href="#cb21-8" aria-hidden="true" tabindex="-1"></a><span class="co"># shape as a function of site/raw material?</span></span>
<span id="cb21-9"><a href="#cb21-9" aria-hidden="true" tabindex="-1"></a><span class="fu">anova</span>(fit.sh)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Analysis of Variance, using Residual Randomization
Permutation procedure: Randomization of null model residuals 
Number of permutations: 10000 
Estimation method: Ordinary Least Squares 
Sums of Squares and Cross-products: Type I 
Effect sizes (Z) based on F distributions

          Df       SS        MS     Rsq      F        Z Pr(&gt;F)
site       1 0.005169 0.0051693 0.02016 0.5761 0.067178 0.4727
Residuals 28 0.251225 0.0089723 0.97984                       
Total     29 0.256394                                         

Call: procD.lm(f1 = shape ~ site, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
</div>
<div class="sourceCode cell-code" id="cb23"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb23-1"><a href="#cb23-1" aria-hidden="true" tabindex="-1"></a><span class="co"># size</span></span>
<span id="cb23-2"><a href="#cb23-2" aria-hidden="true" tabindex="-1"></a>fit.sz <span class="ot">&lt;-</span> <span class="fu">procD.lm</span>(size <span class="sc">~</span> site,</span>
<span id="cb23-3"><a href="#cb23-3" aria-hidden="true" tabindex="-1"></a>                   <span class="at">data =</span> gdf,</span>
<span id="cb23-4"><a href="#cb23-4" aria-hidden="true" tabindex="-1"></a>                   <span class="at">print.progress =</span> <span class="cn">FALSE</span>,</span>
<span id="cb23-5"><a href="#cb23-5" aria-hidden="true" tabindex="-1"></a>                   <span class="at">iter =</span> <span class="dv">9999</span>)</span>
<span id="cb23-6"><a href="#cb23-6" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb23-7"><a href="#cb23-7" aria-hidden="true" tabindex="-1"></a><span class="co"># size as a function of site/raw material?</span></span>
<span id="cb23-8"><a href="#cb23-8" aria-hidden="true" tabindex="-1"></a><span class="fu">anova</span>(fit.sz)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Analysis of Variance, using Residual Randomization
Permutation procedure: Randomization of null model residuals 
Number of permutations: 10000 
Estimation method: Ordinary Least Squares 
Sums of Squares and Cross-products: Type I 
Effect sizes (Z) based on F distributions

          Df     SS      MS     Rsq      F      Z Pr(&gt;F)    
site       1 1458.1 1458.07 0.36213 15.896 2.9469  7e-04 ***
Residuals 28 2568.3   91.73 0.63787                         
Total     29 4026.4                                         
---
Signif. codes:  0 '***' 0.001 '**' 0.01 '*' 0.05 '.' 0.1 ' ' 1

Call: procD.lm(f1 = size ~ site, iter = 9999, data = gdf, print.progress = FALSE)</code></pre>
</div>
</div>
</section>
<section id="morphological-disparity" class="level2">
<h2 class="anchored" data-anchor-id="morphological-disparity">Morphological Disparity</h2>
<p>The trajectory analysis was followed by an analysis of morphological disparity <span class="citation" data-cites="RN11107 RN7041 RN5694">(<a href="#ref-RN11107" role="doc-biblioref">Collyer and Adams 2020</a>; <a href="#ref-RN7041" role="doc-biblioref">Foote 1993</a>; <a href="#ref-RN5694" role="doc-biblioref">Zelditch et al. 2004</a>)</span> to identify potential differences in morphological diversity between size classes.</p>
<div class="cell">
<div class="sourceCode cell-code" id="cb25"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb25-1"><a href="#cb25-1" aria-hidden="true" tabindex="-1"></a><span class="co"># morphological disparity ----</span></span>
<span id="cb25-2"><a href="#cb25-2" aria-hidden="true" tabindex="-1"></a><span class="co"># do any of the groups display greater shape variation among </span></span>
<span id="cb25-3"><a href="#cb25-3" aria-hidden="true" tabindex="-1"></a><span class="co"># individuals relative to the other group?</span></span>
<span id="cb25-4"><a href="#cb25-4" aria-hidden="true" tabindex="-1"></a><span class="fu">morphol.disparity</span>(shape <span class="sc">~</span> site,</span>
<span id="cb25-5"><a href="#cb25-5" aria-hidden="true" tabindex="-1"></a>                  <span class="at">groups =</span> qdata<span class="sc">$</span>site, </span>
<span id="cb25-6"><a href="#cb25-6" aria-hidden="true" tabindex="-1"></a>                  <span class="at">data =</span> gdf, </span>
<span id="cb25-7"><a href="#cb25-7" aria-hidden="true" tabindex="-1"></a>                  <span class="at">print.progress =</span> <span class="cn">FALSE</span>, </span>
<span id="cb25-8"><a href="#cb25-8" aria-hidden="true" tabindex="-1"></a>                  <span class="at">iter =</span> <span class="dv">9999</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
morphol.disparity(f1 = shape ~ site, groups = qdata$site, iter = 9999,  
    data = gdf, print.progress = FALSE) 



Randomized Residual Permutation Procedure Used
10000 Permutations

Procrustes variances for defined groups
  Jowell Farm Millsap Cache 
  0.013009123   0.005690772 


Pairwise absolute differences between variances
              Jowell Farm Millsap Cache
Jowell Farm   0.000000000   0.007318351
Millsap Cache 0.007318351   0.000000000


P-Values
              Jowell Farm Millsap Cache
Jowell Farm        1.0000        0.1032
Millsap Cache      0.1032        1.0000</code></pre>
</div>
<div class="sourceCode cell-code" id="cb27"><pre class="sourceCode r code-with-copy"><code class="sourceCode r"><span id="cb27-1"><a href="#cb27-1" aria-hidden="true" tabindex="-1"></a><span class="co"># do any of the groups display greater size variation among </span></span>
<span id="cb27-2"><a href="#cb27-2" aria-hidden="true" tabindex="-1"></a><span class="co"># individuals relative to the other group?</span></span>
<span id="cb27-3"><a href="#cb27-3" aria-hidden="true" tabindex="-1"></a><span class="fu">morphol.disparity</span>(size <span class="sc">~</span> site,</span>
<span id="cb27-4"><a href="#cb27-4" aria-hidden="true" tabindex="-1"></a>                  <span class="at">groups =</span> qdata<span class="sc">$</span>site, </span>
<span id="cb27-5"><a href="#cb27-5" aria-hidden="true" tabindex="-1"></a>                  <span class="at">data =</span> gdf, </span>
<span id="cb27-6"><a href="#cb27-6" aria-hidden="true" tabindex="-1"></a>                  <span class="at">print.progress =</span> <span class="cn">FALSE</span>, </span>
<span id="cb27-7"><a href="#cb27-7" aria-hidden="true" tabindex="-1"></a>                  <span class="at">iter =</span> <span class="dv">9999</span>)</span></code><button title="Copy to Clipboard" class="code-copy-button"><i class="bi"></i></button></pre></div>
<div class="cell-output cell-output-stdout">
<pre><code>
Call:
morphol.disparity(f1 = size ~ site, groups = qdata$site, iter = 9999,  
    data = gdf, print.progress = FALSE) 



Randomized Residual Permutation Procedure Used
10000 Permutations

Procrustes variances for defined groups
  Jowell Farm Millsap Cache 
      33.4222      115.8262 


Pairwise absolute differences between variances
              Jowell Farm Millsap Cache
Jowell Farm       0.00000      82.40405
Millsap Cache    82.40405       0.00000


P-Values
              Jowell Farm Millsap Cache
Jowell Farm        1.0000        0.1343
Millsap Cache      0.1343        1.0000</code></pre>
</div>
</div>



</section>

<div id="quarto-appendix" class="default"><section class="quarto-appendix-contents" role="doc-bibliography"><h2 class="anchored quarto-appendix-heading">References</h2><div id="refs" class="references csl-bib-body hanging-indent" role="doc-bibliography">
<div id="ref-RN1655" class="csl-entry" role="doc-biblioentry">
Adams, Dean C., and Michael L. Collyer. 2015. <span>“<span class="nocase">Permutation Tests for Phylogenetic Comparative Analyses of High-Dimensional Shape Data: What you Shuffle Matters</span>.”</span> <em>Evolution</em> 69 (3): 823–29. <a href="https://doi.org/10.1111/evo.12596">https://doi.org/10.1111/evo.12596</a>.
</div>
<div id="ref-RN11530" class="csl-entry" role="doc-biblioentry">
Adams, Dean C., Michael L. Collyer, Antigoni Kaliontzopoulou, and Emma Sherratt. 2017. <span>“<span class="nocase">Package geomorph: Geometric Morphometric Analyses of Landmark Data. R package version 3.0.5</span>.”</span> <a href="http://geomorphr.github.io/geomorph/">http://geomorphr.github.io/geomorph/</a>.
</div>
<div id="ref-RN1774" class="csl-entry" role="doc-biblioentry">
Adams, Dean C., and Erik Otarola-Castillo. 2013. <span>“<span class="nocase">geomorph: An R Package for the Collection and Analysis of Geometric Morphometric Shape Data</span>.”</span> <em>Methods in Ecology and Evolution</em> 4 (4): 393–99. <a href="https://doi.org/10.1111/2041-210x.12035">https://doi.org/10.1111/2041-210x.12035</a>.
</div>
<div id="ref-RN8633" class="csl-entry" role="doc-biblioentry">
Adams, Dean C., F. James Rohlf, and Dennis E. Slice. 2013. <span>“A Field Comes of Age: Geometric Morphometrics in the 21st Century.”</span> Journal Article. <em>Hystrix</em> 24 (1): 7–14. <a href="https://doi.org/doi:10.4404/hystrix-24.1-6283">https://doi.org/doi:10.4404/hystrix-24.1-6283</a>.
</div>
<div id="ref-RN1719" class="csl-entry" role="doc-biblioentry">
Anderson, M. J., and C. J. F. Ter Braak. 2003. <span>“<span class="nocase">Permutation Tests for Multi-Factoral Analysis of Variance</span>.”</span> <em>Journal of Statistical Computation and Simulation</em> 73 (2): 85–113. <a href="https://doi.org/10.1080=0094965021000015558">https://doi.org/10.1080=0094965021000015558</a>.
</div>
<div id="ref-RN9565" class="csl-entry" role="doc-biblioentry">
Baken, Erica K., Michael L. Collyer, Antigoni Kaliontzopoulou, and Dean C. Adams. 2021. <span>“<span class="nocase">geomorph v4.0 and gmShiny: Enhanced analytics and a new graphical interface for a comprehensive morphometric experience</span>.”</span> Journal Article. <em>Methods in Ecology and Evolution</em>. <a href="https://doi.org/10.1111/2041-210x.13723">https://doi.org/10.1111/2041-210x.13723</a>.
</div>
<div id="ref-RN11138" class="csl-entry" role="doc-biblioentry">
Bookstein, Fred L. 1986. <span>“Size and Shape Spaces for Landmark Data in Two Dimensions.”</span> Journal Article. <em>Statistical Science</em> 1 (2): 181–242.
</div>
<div id="ref-RN5616" class="csl-entry" role="doc-biblioentry">
———. 1991. <em>Morphometric Tools for Landmark Data: Geometry and Biology</em>. Book. Cambridge: Cambridge University Press.
</div>
<div id="ref-RN8477" class="csl-entry" role="doc-biblioentry">
Bookstein, Fred L., Katrin Schäfer, Hermann Prossinger, Horst Seidler, Martin Fieder, Chris Stringer, Gerhard W. Weber, et al. 1999. <span>“Comparing Frontal Cranial Profiles in Archaic and Modern Homo by Morphometric Analysis.”</span> Journal Article. <em>The Anatomical Record</em> 257 (6): 217–24. https://doi.org/<a href="https://doi.org/10.1002/(SICI)1097-0185(19991215)257:6%3C217::AID-AR7%3E3.0.CO;2-W">https://doi.org/10.1002/(SICI)1097-0185(19991215)257:6%3C217::AID-AR7%3E3.0.CO;2-W</a>.
</div>
<div id="ref-RN11139" class="csl-entry" role="doc-biblioentry">
Chapman, R. E. 1990. <span>“Conventional Procrustes Approaches.”</span> Book Section. In <em>Proceedings of the Michigan Morphometrics Workshop</em>, edited by F. James Rohlf and Fred L. Bookstein, 251–69. Ann Arbor: Special Publication Number 2, The University of Michigan Museum of Zoology.
</div>
<div id="ref-RN11775" class="csl-entry" role="doc-biblioentry">
Collyer, Michael L., and Dean C. Adams. 2018. <span>“<span class="nocase">RRPP: An R Package for Fitting Linear Models to High-Dimensional Data using Residual Randomization</span>.”</span> <em>Methods in Ecology and Evolution</em> 9 (7): 1772–79. https://doi.org/<a href="https://doi.org/10.1111/2041-210X.13029">https://doi.org/10.1111/2041-210X.13029</a>.
</div>
<div id="ref-RN11107" class="csl-entry" role="doc-biblioentry">
———. 2020. <span>“Phylogenetically Aligned Component Analysis.”</span> Journal Article. <em>Methods in Ecology and Evolution</em> 12 (2): 359–72. <a href="https://doi.org/10.1111/2041-210x.13515">https://doi.org/10.1111/2041-210x.13515</a>.
</div>
<div id="ref-RN1756" class="csl-entry" role="doc-biblioentry">
Collyer, Michael L., David J. Sekora, and Dean C. Adams. 2015. <span>“<span class="nocase">A Method for Analysis of Phenotypic Change for Phenotypes Described by High-Dimensional Data</span>.”</span> <em>Heredity</em> 115 (4): 357–65. <a href="https://doi.org/10.1038/hdy.2014.75">https://doi.org/10.1038/hdy.2014.75</a>.
</div>
<div id="ref-RN11141" class="csl-entry" role="doc-biblioentry">
Dryden, I. L., and K. V. Mardia. 1993. <span>“Multivariate Shape Analysis.”</span> Journal Article. <em>Sankhya: The Indian Journal of Statistics, Series A</em> Special Volume 55, Series A, Pt. 3: 460–80.
</div>
<div id="ref-RN11140" class="csl-entry" role="doc-biblioentry">
———. 1998. <em>Statistical Shape Analysis</em>. Book. Chichester: Wiley.
</div>
<div id="ref-RN7041" class="csl-entry" role="doc-biblioentry">
Foote, M. 1993. <span>“Contributions of Individual Taxa to Overall Morphological Disparity.”</span> Journal Article. <em>Paleobiology</em> 19 (4): 403–19.
</div>
<div id="ref-RN1749" class="csl-entry" role="doc-biblioentry">
Goodall, Colin. 1991. <span>“<span class="nocase">Procrustes Methods in the Statistical Analysis of Shape</span>.”</span> <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 53 (2): 285–339.
</div>
<div id="ref-RN11564" class="csl-entry" role="doc-biblioentry">
Gower, J. C. 1975. <span>“<span>Generalized Procrustes Analysis</span>.”</span> <em>Psychometrika</em> 40 (1): 33–51. https://doi.org/<a href="https://doi.org/10.1007/BF02291478">https://doi.org/10.1007/BF02291478</a>.
</div>
<div id="ref-RN7502" class="csl-entry" role="doc-biblioentry">
Gunz, P., P. Mitteroecker, and F. L. Bookstein. 2005. <span>“Semilandmarks in Three Dimensions.”</span> Book Section. In <em>Modern Morphometrics in Physical Anthropology</em>, edited by D. E. Slice, 73–98. New York: Plenum Publishers.
</div>
<div id="ref-RN1746" class="csl-entry" role="doc-biblioentry">
Jolliffe, Ian T. 2002. <em>Principal Component Analysis</em>. New York: Springer.
</div>
<div id="ref-RN11622" class="csl-entry" role="doc-biblioentry">
Kendall, David G. 1981. <span>“<span class="nocase">The Statistics of Shape</span>.”</span> In <em>Interpreting Multivariate Data</em>, edited by V. Barnett, 75–80. New York: Wiley.
</div>
<div id="ref-RN11623" class="csl-entry" role="doc-biblioentry">
———. 1984. <span>“<span class="nocase">Shape Manifolds, Procrustean Metrics, and Complex Projective Spaces</span>.”</span> <em>Bulletin of the London Mathematical Society</em> 16 (2): 81–121. <a href="https://doi.org/10.1112/blms/16.2.81">https://doi.org/10.1112/blms/16.2.81</a>.
</div>
<div id="ref-RN11142" class="csl-entry" role="doc-biblioentry">
Kent, J. T., and K. V. Mardia. 2001. <span>“Shape, Procrustes Tangent Projections and Bilateral Symmetry.”</span> Journal Article. <em>Biometrika</em> 88 (2): 469–85.
</div>
<div id="ref-RN1731" class="csl-entry" role="doc-biblioentry">
Klingenberg, Christian Peter. 2013. <span>“<span class="nocase">Visualizations in Geometric Morphometrics: How to Read and How to Make Graphs Showing Shape Changes</span>.”</span> <em>Hystrix</em> 24 (1): 15–24. https://doi.org/<a href="http://dx.doi.org/10.4404/hystrix-24.1-7691">http://dx.doi.org/10.4404/hystrix-24.1-7691</a>.
</div>
<div id="ref-RN11143" class="csl-entry" role="doc-biblioentry">
O’Higgins, Paul, and Nicholas Jones. 1998. <span>“Facial Growth in Cercocbus Torquatus: An Application of Three-Dimensional Geometric Morphometric Techniques to the Study of Morphological Variation.”</span> Journal Article. <em>The Journal of Anatomy</em> 193 (2): 251–72.
</div>
<div id="ref-RN1646" class="csl-entry" role="doc-biblioentry">
Rohlf, F. James. 1999. <span>“<span class="nocase">Shape Statistics: Procrustes Superimpositions and Tangent Spaces</span>.”</span> <em>Journal of Classification</em> 16 (2): 197–223. <a href="https://doi.org/10.1007/s003579900054">https://doi.org/10.1007/s003579900054</a>.
</div>
<div id="ref-RN7550" class="csl-entry" role="doc-biblioentry">
Rohlf, F. James, and Leslie F. Marcus. 1993. <span>“A Revolution in Morphometrics.”</span> Journal Article. <em>Trends in Ecology &amp; Evolution</em> 8 (4): 129–32. <a href="https://doi.org/10.1016/0169-5347(93)90024-J">https://doi.org/10.1016/0169-5347(93)90024-J</a>.
</div>
<div id="ref-RN478" class="csl-entry" role="doc-biblioentry">
Rohlf, F. James, and Dennis Slice. 1990. <span>“<span class="nocase">Extensions of the Procrustes Method for the Optimal Superimposition of Landmarks</span>.”</span> <em>Systematic Zoology</em> 39 (1): 40–59. <a href="https://doi.org/10.2307/2992207">https://doi.org/10.2307/2992207</a>.
</div>
<div id="ref-RN9551" class="csl-entry" role="doc-biblioentry">
Selden Jr, Robert Z., John E. Dockall, C. Britt Bousman, and Timothy K. Perttula. 2021. <span>“<span class="nocase">Shape as a function of time + raw material + burial context? An exploratory analysis of Perdiz arrow points from the ancestral Caddo area of the American Southeast</span>.”</span> Journal Article. <em>Journal of Archaeological Science: Reports</em> 37: 102916. <a href="https://doi.org/10.1016/j.jasrep.2021.102916">https://doi.org/10.1016/j.jasrep.2021.102916</a>.
</div>
<div id="ref-RN11801" class="csl-entry" role="doc-biblioentry">
Selden Jr., Robert Z. 2018a. <span>“A Preliminary Study of Smithport Plain Bottle Morphology in the Southern Caddo Area.”</span> <em>Bulletin of the Texas Archeological Society</em> 89: 63–89.
</div>
<div id="ref-RN11782" class="csl-entry" role="doc-biblioentry">
———. 2018b. <span>“<span class="nocase">Ceramic Morphological Organisation in the Southern Caddo Area: Quiddity of Shape for Hickory Engraved Bottles</span>.”</span> <em>Journal of Archaeological Science: Reports</em> 21: 884–96. <a href="https://doi.org/10.1016/j.jasrep.2018.08.045">https://doi.org/10.1016/j.jasrep.2018.08.045</a>.
</div>
<div id="ref-RN11716" class="csl-entry" role="doc-biblioentry">
———. 2019. <span>“<span class="nocase">Ceramic Morphological Organisation in the Southern Caddo Area: The Clarence H. Webb Collections</span>.”</span> <em>Journal of Cultural Heritage</em> 35: 41–55. https://doi.org/<a href="https://doi.org/10.1016/j.culher.2018.07.002">https://doi.org/10.1016/j.culher.2018.07.002</a>.
</div>
<div id="ref-RN8312" class="csl-entry" role="doc-biblioentry">
———. 2021. <span>“Louisiana Limitrophe: An Iterative Morphological Exegesis of Caddo Bottle and Biface Production.”</span> Book Section. In <em>Ancestral Caddo Ceramic Traditions</em>, edited by Duncan P. McKinnon, Jeffrey S. Girard, and Timothy K. Perttula, 258–76. Baton Rouge: LSU Press.
</div>
<div id="ref-RN11264" class="csl-entry" role="doc-biblioentry">
———. 2022. <span>“Morphologically Similar, but Regionally Distinct: Perdiz Arrow Points from Caddo Burial Contexts in the American Southeast.”</span> Journal Article. <em>Lithic Technology</em> (in press). <a href="https://doi.org/10.1080/01977261.2022.2095492">https://doi.org/10.1080/01977261.2022.2095492</a>.
</div>
<div id="ref-RN21001" class="csl-entry" role="doc-biblioentry">
Selden Jr., Robert Z., John E. Dockall, and Morgane Dubied. 2020. <span>“A Quantitative Assessment of Intraspecific Morphological Variation in Gahagan Bifaces from the Southern Caddo Area and Central Texas.”</span> <em>Southeastern Archaeology</em> 39 (2): 125–45. <a href="https://doi.org/10.1080/0734578x.2020.1744416">https://doi.org/10.1080/0734578x.2020.1744416</a>.
</div>
<div id="ref-RN11783" class="csl-entry" role="doc-biblioentry">
Selden Jr., Robert Z., John E. Dockall, and Harry J. Shafer. 2018. <span>“<span class="nocase">Lithic Morphological Organisation: Gahagan Bifaces from the Southern Caddo Area</span>.”</span> <em>Digital Applications in Archaeology and Cultural Heritage</em> 10: e00080. <a href="https://doi.org/10.1016/j.daach.2018.e00080">https://doi.org/10.1016/j.daach.2018.e00080</a>.
</div>
<div id="ref-RN479" class="csl-entry" role="doc-biblioentry">
Sherratt, Emma, David J. Gower, Christian Peter Klingenberg, and Mark Wilkinson. 2014. <span>“<span class="nocase">Evolution of Cranial Shape in Caecilians (Amphibia: Gymnophiona)</span>.”</span> <em>Evolutionary Biology</em> 41 (4): 528–45. https://doi.org/<a href="https://doi.org/10.1007/s11692-014-9287-2">https://doi.org/10.1007/s11692-014-9287-2</a>.
</div>
<div id="ref-RN11563" class="csl-entry" role="doc-biblioentry">
Slice, Dennis E. 2001. <span>“<span class="nocase">Landmark Coordinates Aligned by Procrustes Analysis Do Not Lie in Kendall’s Shape Space</span>.”</span> <em>Systematic Biology</em> 50 (1): 141–49. <a href="https://doi.org/10.1080/10635150119110">https://doi.org/10.1080/10635150119110</a>.
</div>
<div id="ref-RN5694" class="csl-entry" role="doc-biblioentry">
Zelditch, Miriam Leah, Donald L. Swiderski, H. David Sheets, and William L. Fink. 2004. <em>Geometric Morphometrics for Biologists : A Primer</em>. Book. Burlington: Elsevier Science. <a href="http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308">http://ebookcentral.proquest.com/lib/tamucs/detail.action?docID=298308</a>.
</div>
</div></section></div></main> <!-- /main -->
<script id="quarto-html-after-body" type="application/javascript">
window.document.addEventListener("DOMContentLoaded", function (event) {
  const toggleBodyColorMode = (bsSheetEl) => {
    const mode = bsSheetEl.getAttribute("data-mode");
    const bodyEl = window.document.querySelector("body");
    if (mode === "dark") {
      bodyEl.classList.add("quarto-dark");
      bodyEl.classList.remove("quarto-light");
    } else {
      bodyEl.classList.add("quarto-light");
      bodyEl.classList.remove("quarto-dark");
    }
  }
  const toggleBodyColorPrimary = () => {
    const bsSheetEl = window.document.querySelector("link#quarto-bootstrap");
    if (bsSheetEl) {
      toggleBodyColorMode(bsSheetEl);
    }
  }
  toggleBodyColorPrimary();  
  const icon = "";
  const anchorJS = new window.AnchorJS();
  anchorJS.options = {
    placement: 'right',
    icon: icon
  };
  anchorJS.add('.anchored');
  const clipboard = new window.ClipboardJS('.code-copy-button', {
    target: function(trigger) {
      return trigger.previousElementSibling;
    }
  });
  clipboard.on('success', function(e) {
    // button target
    const button = e.trigger;
    // don't keep focus
    button.blur();
    // flash "checked"
    button.classList.add('code-copy-button-checked');
    var currentTitle = button.getAttribute("title");
    button.setAttribute("title", "Copied!");
    setTimeout(function() {
      button.setAttribute("title", currentTitle);
      button.classList.remove('code-copy-button-checked');
    }, 1000);
    // clear code selection
    e.clearSelection();
  });
  function tippyHover(el, contentFn) {
    const config = {
      allowHTML: true,
      content: contentFn,
      maxWidth: 500,
      delay: 100,
      arrow: false,
      appendTo: function(el) {
          return el.parentElement;
      },
      interactive: true,
      interactiveBorder: 10,
      theme: 'quarto',
      placement: 'bottom-start'
    };
    window.tippy(el, config); 
  }
  const noterefs = window.document.querySelectorAll('a[role="doc-noteref"]');
  for (var i=0; i<noterefs.length; i++) {
    const ref = noterefs[i];
    tippyHover(ref, function() {
      // use id or data attribute instead here
      let href = ref.getAttribute('data-footnote-href') || ref.getAttribute('href');
      try { href = new URL(href).hash; } catch {}
      const id = href.replace(/^#\/?/, "");
      const note = window.document.getElementById(id);
      return note.innerHTML;
    });
  }
  var bibliorefs = window.document.querySelectorAll('a[role="doc-biblioref"]');
  for (var i=0; i<bibliorefs.length; i++) {
    const ref = bibliorefs[i];
    const cites = ref.parentNode.getAttribute('data-cites').split(' ');
    tippyHover(ref, function() {
      var popup = window.document.createElement('div');
      cites.forEach(function(cite) {
        var citeDiv = window.document.createElement('div');
        citeDiv.classList.add('hanging-indent');
        citeDiv.classList.add('csl-entry');
        var biblioDiv = window.document.getElementById('ref-' + cite);
        if (biblioDiv) {
          citeDiv.innerHTML = biblioDiv.innerHTML;
        }
        popup.appendChild(citeDiv);
      });
      return popup.innerHTML;
    });
  }
});
</script>
</div> <!-- /content -->



</body></html>